app/views/services/index.mobile.haml

```diff
@@ -0,0 +1,15 @@
+-#   Copyright (c) 2010-2011, Diaspora Inc.  This file is
+-#   licensed under the Affero General Public License version 3 or later.  See
+-#   the COPYRIGHT file.
+
+.settings_container.services_page
+  - content_for :page_title do
+    = t('.edit_services')
+
+  = render 'shared/settings_nav'
+
+  .col-md-12
```

Comment:
This is not the correct usage of Bootstrap's grid system. See http://getbootstrap.com/css/#grid
There is some `.container-fluid` missing. You have the same problem on most other pages.